frontend-zs/src/components/projects/Home/Main.js

```diff
@@ -53,8 +54,12 @@ export function ProjectsList() {
         <Container fluid >
             <Row className='ml-md-5 mr-md-5'>
                 {searchAndSortHeader(onSearch, onSort, sortAsc, setIsMapView, isMapView)}
-                {ProjectCardsList(displayProjects, false, history)}
-                {ProjectCardsList(displayProjects, true, history)}
+                {ProjectCardsList(displayProjects, false, isMapView, history)}
+                {isMapView ?
+                    <ProjectsMap id="myMap"
+                        options={{ center: { lat: 48.13, lng: 11.58 }, zoom: 8 }}
+                        projects={displayProjects.map(project => { return { location: project.info?.location.coordinates, title: project.info?.title } })} />
```

Comment:
`project.info?.location.coordinates` might break if a project does not have a location